WalletWasabi.Fluent/Views/Wallets/Home/Tiles/PrivacyBar/PrivacyBar.axaml.cs

```diff
@@ -1,7 +1,7 @@
 using Avalonia.Controls;
 using Avalonia.Markup.Xaml;
 
-namespace WalletWasabi.Fluent.Views.Wallets.Home.Tiles;
+namespace WalletWasabi.Fluent.Views.Wallets.Home.Tiles.PrivacyBar;
```

Comment:
It's PrivacyRing folder

https://github.com/zkSNACKs/WalletWasabi/blob/master/WalletWasabi.Fluent/ViewModels/Wallets/Home/Tiles/PrivacyRing/PrivacyBarViewModel.cs

Reply:
Reverted. Anyways, I think those namespaces are inconsistent in master.